eng/guardian-tools/empty.gdnbaselines

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I assume you don't need this any longer given that the change to use empty paths.